initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -559,26 +559,24 @@ stop_component() {
 
     eval $stopcmd
 
+    unlock ${LOCKPATH}
+    removePid ${PIDPATH}
+
     RESULT=0
-    local counter=0
-    while [[ $RESULT -eq 0 && $counter -le 20 ]]; do
+    local waittime=30
```

Comment:
Well..
If it gives up and logs failed.. what does it say?
And what happens if you try to start after this failed stop?

Our production system can take > 10 mins to stop (Dali saves the entire store when exiting).